conformance/operator_test.go

```diff
@@ -275,10 +276,16 @@ var _ = Describe("operator", func() {
 		Context("VF flags", func() {
 			debugPod := &corev1.Pod{}
 			intf := &sriovv1.InterfaceExt{}
+			numVfs := 5
 
 			validationFunction := func(networks []string, containsFunc func(line string) bool) {
+				// Validate all the virtual functions are in the host namespace
+				_, err := findPodVFInHost(intf.Name, numVfs, debugPod)
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).To(Equal("failed to find the vf number that was moved into the pod"))
```

Comment:
This error is more like: expecting all the vfs to be in the host namespace, found a missing one.

Reply:
misread it, it's fine